docs/pages/includes/database-access/database-config.yaml

```
@@ -2,7 +2,13 @@ db_service:
   # Enables the Database Service.
   enabled: "yes"
 
-  # Matchers for database resources created with "tctl create" command.
+  # Matchers for database resources created with "tctl create" command or by the
+  # discovery service.
+  #
+  # All database resources have a predefined "teleport.dev/origin" label with
+  # one of the following values:
+  # "dynamic": database resources created with "tctl create" command
+  # "cloud": database resources created by the discovery service
```

What about resources manually defined in a config file? Do they get a different or no label?

---

Ya, this does feel a little confusing.

So technically, there are total 3 values for the origin label for databases: "cloud", "dynamic", and config".

Both "dynamic" and "cloud" are the "resources" that can be received by using this dynamic resources section. They are all dynamic resources just "cloud" is used for dynamic resources created by the discovery service.

Databases auto-discovered by this database agent using "aws", "azure" sections in this config also has "cloud" label.

"static" databases in the "databases" section has the "config" label.

Any suggestions how to make this clear?

---

To avoid confusion, I think it's important that we're careful about how we use these terms. For example:

> So technically, there are total 3 values for the origin label for databases: "cloud", "dynamic", and "config".

And:
> "static" databases in the "databases" section has the "config" label.

These sentences sort of muddy the waters on what a "label" is.

I get what you're saying, I just want to point out that part of what makes this a confusing subject is that we really need to adhere to the right terminology.

Maybe we could just add another line to this section:

```suggestion
  # All database resources have a predefined "teleport.dev/origin" label with
  # one of the following values:
  # "dynamic": database resources created with "tctl create" command
  # "cloud": database resources created by the discovery service
  # "config": database resources defined in the "databases" array below
```